src/Bridges/indicatorbridge.jl

```diff
@@ -36,13 +36,13 @@ function bridge_constraint(::Type{<:IndicatorActiveOnFalseBridge}, model::MOI.Mo
         f.constants
     )
     ci = MOI.add_constraint(model, f2, MOI.IndicatorSet{MOI.ACTIVATE_ON_ONE}(s.set))
-
-    return IndicatorActiveOnFalseBridge{T, MOI.VectorAffineFunction{T}, S}(z2, zo_cons, dcons, ci)
+    BT = concrete_bridge_type(IndicatorActiveOnFalseBridge{T}, typeof(f), typeof(s))
```

Comment:
`concrete_bridge_type` shouldn't be called here, the first argument of `bridge_constraint` should already be concrete

Reply:
See https://github.com/JuliaOpt/MathOptInterface.jl/pull/752

Reply:
fixed in last commits

Reply:
This line should still be removed, `BT` is simply the first argument